src/HelixMarkdownPreview.js

```diff
@@ -165,15 +165,15 @@ if (typeof window.HelixMarkdownPreview === 'undefined') {
       }
 
       /**
-       * Removes the preview window and resets the extension.
+       * Removes the preview window closes the popup and resets the extension.
        * @private
        */
       function removePopup() {
         if (popup) {
           delete popup.init;
           popup.close();
         }
-        chrome.extension.getViews({ type: 'tab' }).forEach((v) => {
+        chrome.tabs.remove({ type: 'tab' }).forEach((v) => {
```

Comment:
I don't think you are using [`chrome.tabs.remove`](https://developer.chrome.com/extensions/tabs#method-remove) correctly. This change disables the entire feature and throws the following error in the background page:
```
_generated_background_page.html:1 Error handling response: TypeError: Error in invocation of tabs.remove([integer|array] tabIds, optional function callback): No matching signature.
    at removePopup (chrome-extension://apjaifekenphoibpleklnaohagpcnnhg/HelixMarkdownPreview.js:176:21)
    at createPopup (chrome-extension://apjaifekenphoibpleklnaohagpcnnhg/HelixMarkdownPreview.js:190:9)
    at Object.start (chrome-extension://apjaifekenphoibpleklnaohagpcnnhg/HelixMarkdownPreview.js:338:11)
    at chrome-extension://apjaifekenphoibpleklnaohagpcnnhg/background.js:75:20
```
Please test your fix thoroughly to make sure it doesn't break existing functionality.

Reply:
I´m short on time right now but I´ll keep trying to fix this. First I thought that changing the type to 'popup' at getViews would be an easy fix... my bad. Then I thought that using tabs.remove would do the trick. I´ll do that. Sorry about the time you took to review it. I never saw this kind of behaviour on a chrome extension before so I´d gladly keep working on to solve this.